spec/breadcrumbs/breadcrumb_spec.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+require 'bugsnag/breadcrumbs/breadcrumb'
+
+RSpec.describe Bugsnag::Breadcrumbs::Breadcrumb do
+  describe "#message" do
+    it "is assigned in #initialize" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new("my message", nil, nil, nil)
+
+      expect(breadcrumb.message).to eq("my message")
+    end
+  end
+
+  describe "#type" do
+    it "is assigned in #initialize" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new(nil, "test type", nil, nil)
+
+      expect(breadcrumb.type).to eq("test type")
+    end
+  end
+
+  describe "#meta_data" do
+    it "is assigned in #initialize" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new(nil, nil, {:a => 1, :b => 2}, nil)
+
+      expect(breadcrumb.meta_data).to eq({:a => 1, :b => 2})
+    end
+  end
+
+  describe "#auto" do
+    it "defaults to false" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new(nil, nil, nil, nil)
+
+      expect(breadcrumb.auto).to eq(false)
+    end
+
+    it "is true if auto argument == :auto" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new(nil, nil, nil, :auto)
+
+      expect(breadcrumb.auto).to eq(true)
+    end
+
+    it "if false if auto argument is anything else" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new(nil, nil, nil, :manual)
+
+      expect(breadcrumb.auto).to eq(false)
+    end
+  end
+
+  describe "#timestamp" do
+    it "is stored as a timestamp" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new(nil, nil, nil, nil)
+
+      expect(breadcrumb.timestamp).to be_within(0.5.second).of Time.now
+    end
+  end
+
+  describe "#ignore" do
```

Comment:
This should be `#ignore?` (to match the actual method name called by all examples under this describe block)